.buildkite/pipeline.yml

```diff
@@ -953,6 +953,16 @@ steps:
       slurm_ntasks: 1
       slurm_gres: "gpu:1"
 
+  - label: "gpu_bomex_edmf"
+    key: "gpu_bomex_edmf"
+    command:
+      - "mpiexec julia --color=yes --project=docs/ test/Atmos/EDMF/bomex_edmf.jl --diagnostics=default --fix-rng-seed"
```

Comment:
I think the use of --project=docs/ here might be a typo?  `--project`

Reply:
Ah, I thought I had fixed this. Will change, thanks!